docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -357,18 +444,20 @@ Configurations, Producer PropertiesProducer Properties, and Advanced Producer Co
 |===
 |Properties | Type |Description
 
-|*spring.cloud.stream.servicebus*.bindings.<binding-name>.consumer.requeueRejected |
-boolean | xxx
-|*spring.cloud.stream.servicebus*.bindings.<binding-name>.consumer.checkpointMode| CheckpointMode
-| xxx
-|*spring.cloud.stream.servicebus*.bindings.<binding-name>.consumer.maxConcurrentCalls | Integer |
-|*spring.cloud.stream.servicebus*.bindings.<binding-name>.consumer.maxConcurrentSessions | Integer |
+|*spring.cloud.stream.servicebus*.bindings.<binding-name>.consumer.requeue-rejected |If the
+failed messages are routed to the DLQ.
+boolean |
+|*spring.cloud.stream.servicebus*.bindings.<binding-name>.consumer.checkpoint-mode| CheckpointMode
+| The checkpoint mode of checkpointing message.
+|*spring.cloud.stream.servicebus*.bindings.<binding-name>.consumer.max-concurrent-calls | Integer
```

Comment:
this entry and the below ones could be omitted here, it should be covered in "Advanced Consumer Configuration"